types/prettier/doc.d.ts

```diff
@@ -81,26 +101,62 @@ export namespace builders {
         placeholder: symbol;
     }
 
+    interface GroupOptions {
+        shouldBreak?: boolean;
+        id?: symbol;
+    }
+
     function addAlignmentToDoc(doc: Doc, size: number, tabWidth: number): Doc;
-    function align(n: Align['n'], contents: Doc): Align;
+    /** @see [align](https://github.com/prettier/prettier/blob/main/commands.md#align) */
+    function align(widthOrString: number | string, doc: Doc): Align;
```

Comment:
This is wrong, `number | string | { type: 'root' };` is correct, I run into this few days ago.